specs/binding-mqtt-kafka.spec/src/test/java/io/aklivity/zilla/specs/binding/mqtt/kafka/config/SchemaTest.java

```diff
@@ -48,4 +48,12 @@ public void shouldValidateProxyWithOptions()
 
         assertThat(config, not(nullValue()));
     }
+
+    @Test
+    public void shouldValidateProxyWhenMqttTopicWithKafkaTopic()
```

Comment:
This might simplify to `shouldValidateProxyWhenTopicWithMessages` if we rename `with` `topic` to `messages` as mentioned above.